models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -188,15 +188,15 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
     output[[10]] <- (sub.sipnet.output$plantWoodC * 0.001) + (sub.sipnet.output$plantLeafC * 0.001) + 
       (sub.sipnet.output$coarseRootC + sub.sipnet.output$fineRootC) * 0.001 # Total living C kgC/m2
     output[[11]] <- (sub.sipnet.output$soil * 0.001) + (sub.sipnet.output$litter * 0.001)  # Total soil C kgC/m2
-    if (revision == "r136") {
-      output[[12]] <- (sub.sipnet.output$evapotranspiration * 10 * PEcAn.data.atmosphere::get.lv()) / timestep.s  # Qle W/m2
-    } else {
+    if (revision == "unk") {
```

Comment:
This looks like an important bugfix that deserves a CHANGELOG entry so folks know it's there.

Reply:
Yes this is a super important fix, before the LE model output would be 10^3 off observations. I will add a CHANGELOG entry.